app/Http/Services/Contents/It/Event/ConnpassEventService.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Services\Contents\It\Event;
+
+/**
+ * ConnpassEventService.
+ *
+ * ConnpassEventを扱うService
+ */
+class ConnpassEventService
+{
+    /**
+     * Create a new instance.
+     */
+    public function __construct()
+    {
+    }
+    
+    /**
+     * ConnpassAPIにリクエストするURLの生成
+     * @param リクエストパラム
+     * @return string ConnpassAPIにリクエストするURL
+     */
+    private function _createRequestConnpassApiUrl($param) {
+        $year = $param['year'];
+        $month = $param['month'];
+        $count = $param['count'];
+        $firstMonth = date("$year-$month-01");
+        $lastMonth = date("$year-$month-t");
+
+        return "http://connpass.com/api/v1/event/?keyword=python";
+    }
+
+    /**
+     * connpassのAPIからJSONを取得する.
+     * 
+     * @param array リクエストパラム
+     * @return array connpassのイベントを取得
+     */
+    public function getConnpassApiOneMonthEventJson($param)
```

Comment:
__[phpmd]__
* The variable $status_code is not named in camelCase.
* The variable $status_code is not named in camelCase.